resources/rpm/project.spec.erb

```diff
@@ -14,6 +14,16 @@
     /usr/lib64/rpm/brp-strip-static-archive %{__strip} \
     /usr/lib64/rpm/brp-strip-comment-note %{__strip} %{__objdump} \
 %{nil}
+<%- elsif !@platform.is_aix? -%>
```

Comment:
Do you need to check for cisco-wrlinux here also?

Reply:
nope, see the start of the if statement https://github.com/puppetlabs/vanagon/pull/522/files#diff-a92f582d860af1e32c4e5b9e8f9bb177R3

Reply:
ahh 👍